README.md

```diff
@@ -42,6 +42,7 @@ Returns a Promise of the image URI.
     - `"base64"`: encode as base64 and returns the raw string. Use only with small images as this may result of lags (the string is sent over the bridge). *N.B. This is not a data uri, use `data-uri` instead*.
     - `"data-uri"`: same as `base64` but also includes the [Data URI scheme](https://en.wikipedia.org/wiki/Data_URI_scheme) header.
  - **`filename`** *(string)*: the name of the generated file if any (Android only). Defaults to `ReactNative_snapshot_image_${timestamp}`.
+ - **`isScrollView`** *(bool)*: View is evaluated the real height when the value is true.
```

Comment:
yeah, I also think that. isScrollView is not good ,but i can't think other one. i am english is very bad.
and then, i can't implement it in iOS, because i can't coding swift,. i will try iOS but not just yet